shared/actions/wallets.js

```diff
@@ -60,6 +66,7 @@ function* loadEverything(action: WalletsGen.LoadEverythingPayload) {
 
 function* walletsSaga(): Saga.SagaGenerator<any, any> {
   yield Saga.safeTakeEveryPure(WalletsGen.loadAccounts, loadAccounts, loadAccountsSuccess)
+  yield Saga.safeTakeEveryPure(WalletsGen.selectAccount, loadAccount)
```

Comment:
i think we want to invert this relationship.

We want to have an array of actions which lead to a single side effect. We don't want to have an action which spawns a bunch of actions

aka

we should prefer
```
yield Saga.safeTakeEveryPure(WalletsGen.selectAccount, loadAssets)
yield Saga.safeTakeEveryPure(WalletsGen.selectAccount, loadPayments)
```

then if some other thing causes a loadAssets, like a call from the server saying assets changed we'd do
```
yield Saga.safeTakeEveryPure([WalletsGen.selectAccount, WalletsGen.recievedAssetsUpdatedNotification], loadAssets)
```

I think this leads to less logical flows and less branching. There's one place where loadAssets actually happens and the relationships are more clear.

Let me know if the above makes sense. @keybase/react-hackers this ^ is something we should all grok